go.mod

```diff
@@ -1,18 +1,19 @@
-module github.com/developer-guy/cosign-gatekeeper-provider
+module github.com/sigstore/cosign-gatekeeper-provider
 
 go 1.17
 
 require (
+	get.porter.sh/porter v0.38.8
```

Comment:
Are you sure to get the port lib in for the helper function in mage?

Reply:
not 100% maybe i will drop that and reimplement the git variables locally, I was re-thinking on that and maybe this brings a lot of things that we will not use.
thanks for the heads up

Reply:
i've dropped that for now and when we implement the version or other things that need ldflags I will reimplement the feature.

Reply:
PTAL @dirien

Reply:
cool!